components/email/src/Email.svelte

```diff
@@ -587,22 +587,27 @@
     const existingDraft = activeThread?.drafts?.find(
       (draft) => draft.reply_to_message_id === message.id,
     );
+    const subject = `Fwd: ${message.subject}`;
+    const value = {
+      reply_to_message_id: message.id,
+      subject: subject,
+      body: message.body,
+    };
     if (existingDraft) {
-      dispatchDraftEvent(event, existingDraft);
+      value.body = existingDraft.body;
+      dispatchEvent("forwardClicked", {
+        event,
+        message: { ...existingDraft, subject },
```

Comment:
how can you forward a draft? Isn't that just sending it as a message?

Since you're dispatching exactly the same object, can you add `const messageType = getMessageType(activeThread);` and change this so `message` can be either a draft or message

Reply:
Hi @heathergh , this code here only dispatches the `forwardClicked` event with the `message` in the detail. 
In order to identify the type of the `message` passed, the Nylas Message has an `object` property which will tell the type of object, _draft_ or _message_, when listening to the event.

I try to understand your question here, so please correct me if there's any concerns.

Reply:
Hi @heathergh Sorry about the confusing code here.

The code right here that fetches and includes body in the event, is for cases when user trying to reply/forward a message in a thread, and there already exists a draft in thread that's reply/forward the selected message. 

In this case, we only toggle open that draft, instead of creating a new message with new message body for forward or replying. It's essentially the same as clicking on that draft message.

It's actually the same behaviour as how Gmail treats its draft messages, since most of our features are following Gmail's behaviour:
https://user-images.githubusercontent.com/14408339/160185862-93c6f053-66cc-4c30-bf82-7dbd8a88d81c.mov

Reply:
interesting! I was not aware of that. Thanks for taking the time to make a recording :)